tests/conftest.py

```
@@ -63,7 +64,151 @@ def refresh_databases() -> None:
+                ("UPDATE cost_invest SET units = 'dollar * meter' WHERE ROWID = 1", ()),
+            ],
+        ),
+    ]
+
+    for db_name, modifications in unit_test_variations:
+        output_path = test_output_dir / db_name
+        create_unit_test_db_from_source(source_db, output_path, modifications)
+        print(f'Created unit test DB: {db_name}')
+
+
+def pytest_configure(config: Config) -> None:
```

Ruff (0.14.8)

Unused function argument: `config`

(ARG001)